controls/views.py

```diff
@@ -220,6 +220,10 @@ def rename_element(request,element_id):
     try:
         new_name = request.POST.get("name", "").strip() or None
         new_description = request.POST.get("description", "").strip() or None
+
+        if Element.objects.filter(name=new_name).exists() is True:
```

Comment:
Could just have:
`if Element.objects.filter(name=new_name).exists():`

Without `is True`